content/pt-br/kubernetes.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Kubernetes
+status: Completed
+category: tecnologia
+tags: ["infraestrutura", "fundamental", ""]
+---
+
+## O que é
+
+Kubernetes, normalmente abreviado como K8s, é um orquestrador de contêineres de código aberto. 
+Ele automatiza o ciclo de vida de aplicações em containers em infraestruturas modernas, funciona como um "sistema operacional de datacenter" que gerencia aplicativos em um [sistema distribuído](/pt-br/distributed-systems/).
```

Comment:
Remove /pt-br/ /distributed-systems/ link. In this case we dont't have any translation into Portuguese

Reply:
Same sugestion as self-healing.